client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -170,11 +171,13 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.FRONT_PAGE_UPDATED:
 			taskData = {
 				timing: 20,
-				title: translate( 'Update your homepage' ),
+				title: isUsingSiteEditor
+					? translate( "Update your site's design" )
+					: translate( 'Update your homepage' ),
 				description: translate(
 					"We've created the basics, now it's time for you to update the images and text. Make a great first impression. Everything you do can be changed anytime."
 				),
-				actionText: translate( 'Edit homepage' ),
+				actionText: isUsingSiteEditor ? translate( 'Edit Site' ) : translate( 'Edit homepage' ),
```

Comment:
Thanks for chiming in! Happy to update this to sentence case if so.

Reply:
Mind adding some tests?

Reply:
> I believe all Calypso buttons are supposed to be sentence-case ("Edit site"), not title-case – am I correct @sixhours ?

I remember this to be the case as well, however much I may personally find it awkward. 😆 

Since the original copy "Edit homepage" followed sentence case, I think it would make sense to follow that standard here with "Edit site" as well. We can always change it back to title case later if requested.